src/pages/studyView/charts/ChartContainer.tsx

```diff
@@ -103,7 +103,7 @@ export class ChartContainer extends React.Component<IChartContainerProps, {}> {
     }
 
     @computed get data(){
-        return this.props.promise.result
+        return _.isUndefined(this.props.promise.result) ? [] : this.props.promise.result;
```

Comment:
this.props.promise.result || []

Reply:
also, it is possible in mobxpromise to make a default result.   not sure if you can do it here if you're using cache but worth a look.  not a big deal either way.